ui/src/app/pages/data/data-set/data-set-component.tsx

```diff
@@ -1104,6 +1108,14 @@ export const DataSetComponent = fp.flow(withUserProfile(), withCurrentWorkspace(
       </div>;
     }
 
+    onClickExport() {
+      if (this.state.selectedDomains.has(Domain.WHOLEGENOMEVARIANT)) {
```

Comment:
nit: technically to avoid race conditions here when setting up a cross-state  dependency like this, I believe it would be better to do:

```
this.setState((state, props) => {
  if state.selectedDomains.has(...) {
    return ...
  }
  return ...
})
```

Reply:
good call, I wasn't aware of that interaction.